src/gluonts/mx/block/regularization.py

```diff
@@ -0,0 +1,179 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# Standard library imports
+from typing import Optional, List
+
+# Third-party imports
+from mxnet.gluon.loss import Loss
+
+# First-party imports
+from gluonts.core.component import validated
+from gluonts.model.common import Tensor
+
+
+class ActivationRegularizationLoss(Loss):
+    r"""Computes Activation Regularization Loss. (alias: AR)
+    The formulation is as below:
+    .. math::
+    L = \alpha L_2(h_t)
+    where :math:`L_2(\cdot) = {||\cdot||}_2, h_t` is the output of the RNN at timestep t.
+    :math:`\alpha` is scaling coefficient.
+    The implementation follows the work::
+    @article{merity2017revisiting,
+    title={Revisiting Activation Regularization for Language RNNs},
+    author={Merity, Stephen and McCann, Bryan and Socher, Richard},
+    journal={arXiv preprint arXiv:1708.01009},
+    year={2017}
+    }
```

Comment:
We put references in [bibliography.rst](https://github.com/awslabs/gluon-ts/blob/master/docs/bibliography.rst) and cite them here. Check [docstring](https://github.com/awslabs/gluon-ts/blob/master/src/gluonts/model/deepar/_estimator.py#L62) in DeepAR estimator.